core/services/form/meta/inputs/Input.php

```diff
@@ -10,6 +10,11 @@ class Input
      */
     public const TYPE_CHECKBOX = 'checkbox';
 
+    /**
+     * indicates that the HTML input type is 'multi checkbox'
+     */
+    public const TYPE_CHECKBOX_MULTI = 'checkbox-multi';
```

Comment:
what's `checkbox-multi` and how is that different than using multiple `checkbox` inputs?

Reply:
That's actually a good question. Checkbox should ideally be multiple while as single checkbox should be replaced by `Switch`

Reply:
💯% agree